release/golden_notebook_tests/dask_xgboost_app_config.yaml

```diff
@@ -17,3 +17,8 @@ post_build_cmds:
   - pip uninstall -y ray || true
   - pip install -U {{ env["RAY_WHEELS"] | default("ray") }}
   - {{ env["RAY_WHEELS_SANITY_CHECK"] | default("echo No Ray wheels sanity check") }}
+  - # Upgrade the XGBoost-Ray version in post build commands, otherwise it will be cached in the Anyscale Docker image.
+  - echo {{ env["TIMESTAMP"] }}
+  - pip install -U xgboost # Upgrade to latest xgboost version so cached version is not used.
+  - pip uninstall xgboost_ray -y # Uninstall first so pip does a reinstall.
+  - pip install -U --no-cache-dir git+https://github.com/ray-project/xgboost_ray.git#egg=xgboost_ray
```

Comment:
Remove this from `pip_packages` now that we're doing this?

Reply:
It's needed in pip_packages to still install in the driver.